dictionary/lib/dictionary/server.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Dictionary.Server do
+
```

Comment:
# -10

It took me quite a while to see what you are doing here. 

The idea was to keep the dictionary state (the word list) in a separate, long-running process (the agent). You can then call that process to get a random word.

But you've added a second layer of process on top of this. That's a lot of added complexity, which made both of our lives harder, and made the program less reliable, simply because there's more code to go wrong.